index.html

```diff
@@ -545,14 +547,26 @@ <h4>The MediaStreamTrackAudioStats interface</h4>
               such as if the track is muted or disabled, then the counters do
               not increase.</p>
             </div>
+            <li>
+              <p>The <dfn data-lt="current audio latency">current audio
```

Comment:
> Define what a certain concept is [...] Don't use the word "note" [...] say what needs to be done, not what doesn't need to be done

Reworded. If this is not to your liking please suggest an edit that is.

I now have a definition of "input latency" which can be referenced by both the "latest input latency" (this PR) and for min/max/avg latency (future PR being discussed in #128).

> In our case, it's the time, in milliseconds, between the point in time an audio input device has acquired a signal, and the time it is able to provide it to an output device.

The sink may not be an output device. But I reworded it to use your language except I say "to delivery of any of its sinks". Does that make sense to you?

> Similarly, we could spec the second part as the point in time the audio data is available to script.

Not sure how to address this comment. Isn't "the script" just another sink?

> Then say that it can be unavailable. Do we really want a nullable here, forcing authors to use an if statement (or something like that)?

I agree it would be nice to avoid "if not null" since it would only really be null before the first frames have been delivered. I guess it could be up to the user agent to make a sensible initial guess? I don't know, I removed nullable for API ergonomics reasons, but it's a bit under-specified what should happen if you call `track.stats.latency` inside the `getUserMedia` promise's `resolve()` in which case I don't know if a measurement has necessarily been made yet, depending on implementation?